Tools/CISupport/ews-build/steps.py

```diff
@@ -2837,8 +2842,10 @@ def send_email_for_new_build_failure(self):
 
             if patch_id and not self.should_send_email_for_patch(patch_id):
                 return
-            if pr_number and not self.should_send_email_for_pr(pr_number, self.getProperty('repository')):
-                return
+            if pr_number:
+                should_send_email = yield self.should_send_email_for_pr(pr_number, self.getProperty('repository'))
+                if not should_send_email:
+                    return
```

Comment:
Do we need defer.returnValue(None) in all the returns inside this method (now that it has `@defer.inlineCallbacks`)?

Reply:
The way I like to think about this is that `defer.inlineCallbacks` turns every `return` into a `break`. We don't generally need to do `defer.returnValue(None)` because that's the default of `defer.inlineCallbacks` (similar to how these three statements are identical in Python)
```
def f_a():
    pass

def f_b():
    return

def f_c():
    return None
```